request_ssh_access/__main__.py

```diff
@@ -18,26 +18,43 @@
 
 
 def main(args=None):
+    class bcolors:
+        HEADER = "\033[95m"
+        OKBLUE = "\033[94m"
+        OKGREEN = "\033[92m"
+        PROMPT = "\033[93m"
+        FAIL = "\033[91m"
+        ENDC = "\033[0m"
+        BOLD = "\033[1m"
+        UNDERLINE = "\033[4m"
+
     if args is None:
         args = sys.argv[1:]
     args = parse_args(args)
 
-    if args.ttl:
-        if args.ttl > config.MAX_TTL:
-            raise Exception("WARNING: The TTL requested is greater than MAX_TTL")
+    if args.ttl > config.MAX_TTL:
+        sys.tracebacklimit = -1
```

Comment:
instead of this, we should wrap the main() call with try/except. if there was an exception, just print out and exit(1):

```python
if __name__ == '__main__':
  try:
     exit(main())
  except Exception as e:
    print(e)
    exit(1)
```

Reply:
whenever you encounter an unrecoverable exception in the code, you could just raise, and the "top-level" exception handler will take care of pretty printing to the user.

Reply:
Thanks @adamn-hmrc will update this now for you :+1:

Reply:
With that change I would have to add more tests :cry: 
FAIL Required test coverage of 90% not reached. Total coverage: 89.13%

Reply:
Commit added :+1: